src/k8s/pkg/utils/dqlite/k8s-dqlite.go

```diff
@@ -0,0 +1,40 @@
+package dqlite
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/canonical/go-dqlite/client"
+	"github.com/canonical/k8s/pkg/snap"
+)
+
+var (
+	// TODO(bschimke): add the port as a configuration option to k8sd so that this can be determined dynamically.
+	K8sDqliteDefaultPort = 9000
+)
+
+// GetK8sDqliteClusterMembers queries the local k8s-dqlite datastore for its members.
+//
+// TODO:
+// This should be done by using the go-dqlite client implementation.
+// However, when I tried to use it the client connects, but returns an empty cluster member list.
+func GetK8sDqliteClusterMembers(ctx context.Context, snap snap.Snap) ([]NodeInfo, error) {
+	c, err := client.DefaultNodeStore(snap.CommonPath("/var/lib/k8s-dqlite/cluster.yaml"))
```

Comment:
Not sure if I'm missing something, isn't this still using the go-dqlite client?

Reply:
Yes, the comment is a bit misleading.
The client has a `Client` and `NodeStore`. Client is a "real" client to perform actions on the cluster (remove, add etc.) while node store simply parses the cluster informations and has no additional functionality. 
I decided to keep this as otherwise I'd need to capture and parse the output from the dqlite wrapper call which seems error prone and clumpsy.